packages/core/services/google-maps-types.ts

```diff
@@ -435,16 +435,18 @@ export interface KmlMouseEvent extends MouseEvent {
   pixelOffset: Size;
 }
 
-export interface TransitLayer extends MVCObject {
```

Comment:
Google maps has TransitLayer and BicycleLayer classes, but not MapLayer class.

Reply:
Why don't you just split out the classes

Reply:
keeping one LayerManager, though, is a good idea, I just think that at the level of google apis, it should be separate since google keeps it separate